micrometer-jmx/src/main/scala/com/avast/sst/micrometer/jmx/MicrometerJmxModule.scala

```diff
@@ -54,7 +54,7 @@ object MicrometerJmxModule {
     override val step: Duration = Duration.ofMillis(c.step.toMillis)
 
     // the method is @Nullable and we don't need to implement it here
-    @SuppressWarnings(Array("org.wartremover.warts.Null"))
+    @SuppressWarnings(Array("org.wartremover.warts.Null", "scalafix:DisableSyntax.null"))
```

Comment:
Why don't we only use Scalafix for this? It's very unfortunate that you have to disable two different rules. Imagine somone not knowing this... you disable WartRemover and it still does not compile so you have to go back and disable also Scalafix. I think every rule should be checked by only single tool.

Reply:
Is enabling all the desirable features of Scalafix and disabling WartRemover in one PR what you guys are calling for?

Or more gradual approach, with multiple PR, but keeping no overlap what each tool enforces/covers?

Reply:
I don't think it's really important. If you enable null checking in Scalafix you should probably disable it in WartRemover in the same PR.